spec/lilypond_spec.rb

```diff
@@ -4,10 +4,10 @@
 \\relative c' { f d f a d f e d cis a cis e a g f e }
 eos
 
-describe Asciidoctor::Diagram::LilypondBlockMacroProcessor, :broken_on_windows, :broken_on_osx do
+describe Asciidoctor::Diagram::LilypondBlockMacroProcessor, :broken_on_osx, :broken_on_windows do
```

Comment:
I think you want to remove `broken_on_osx` here

Reply:
@pepijnve lilypond is still a work in progress atm.

Reply:
I can remove the section from the workflow or comment it out while I work on it.

Reply:
Ok, no problem. Lilypond is a bit of a pain on macOS at the moment indeed due to the lack of 64-bit binaries.

Reply:
I wont let it defeat me @pepijnve !

Reply:
That's the spirit! 💪